### ISSUE TITLE
Provide AppleScript Interface (to enable integration)

### ISSUE THREAD

**Issue by maxim** [topfunky/PeepOpen-Issues#219]
If there was an ability to tell PeepOpen to reindex through an AppleScript command, it'd be possible to integrate with rails generators, file-change watchers, etc.

**Comment by topfunky** [comment]
Good idea. I've looked at AppleScript integration and it looks pretty tricky, but I'd like to figure it out.
